srcpkgs/ndctl/template

```diff
@@ -1,28 +1,28 @@
 # Template file for 'ndctl'
 pkgname=ndctl
-version=70.1
+version=78
 revision=1
-build_style=gnu-configure
-configure_args="--disable-docs --without-systemd"
-hostmakedepends="automake libtool pkg-config xmlto"
+build_style=meson
+configure_args="-Dsystemd=disabled"
+hostmakedepends="pkg-config xmlto ruby-asciidoctor"
 makedepends="eudev-libudev-devel json-c-devel libkmod-devel libuuid-devel
- bash-completion keyutils-devel"
+ bash-completion keyutils-devel libtraceevent-devel libtracefs-devel iniparser-devel"
 short_desc="Utility library for managing libnvdimm"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-only, GPL-2.0-only"
 homepage="https://github.com/pmem/ndctl"
+changelog="https://github.com/pmem/ndctl/releases"
 distfiles="https://github.com/pmem/ndctl/archive/v${version}.tar.gz"
-checksum=2b2a7e6228481076dc807df7a658ba91017658fcb932e8bea489330380d2d682
-conf_files="/etc/ndctl/monitor.conf"
+checksum=80596932920a3eb42551fc0d978f22bfa6a620f57af60c898dc0d0e303c086a5
+conf_files="/etc/ndctl.conf.d/monitor.conf
+ /etc/ndctl.conf.d/ndctl.conf
+ /etc/ndctl/keys/keys.readme
+ /etc/daxctl.conf.d/daxctl.example.conf"
```

Comment:
Having these two files in `conf_files` doesn't seem right to me, what do you think?